API/Mappers/MapperProfile.cs

```diff
@@ -42,7 +42,8 @@ public MapperProfile()
                             (int)orderItemDto.Amount!,
                             (int)orderItemDto.Index!));
                         var orderItems = context.Mapper.Map<IEnumerable<OrderItem>>(orderDto.ProductOrder!.OrderItems);
-                        var productOrder = new ProductOrder((Guid)orderDto.Id!, orderItems);
+                        var productOrder = new ProductOrder((Guid)orderDto.Id!) { OrderItems = orderItems };
```

Comment:
maciau is ProductOrder constructoriaus isemei tuos orderItems, tai gali palikt, ir cia pakviest pilnai konstruktoriu tsg, ar kazkas neveikia jei taip padarai?

Reply:
neveikia. nepraeina migracija